server.js

```diff
@@ -24,10 +14,54 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hello')
+})
+
+// Returns: All data from netflix  
+app.get('/titles', (req, res) => {
+  const page = req.query.page ?? 0
+  const pageSize = req.query.pageSize ?? 20
+  const startIndex = page * pageSize
+  const endIndex = startIndex + +pageSize
+  const titlesPerPage = netflixData.slice(startIndex, endIndex)
+  const returnObject = {
+    pageSize: pageSize,
+    page: page,
+    maxPages: parseInt(netflixData.length / pageSize),
+    results: titlesPerPage
+  }
+  res.send(returnObject)
+})
+
+app.get('/titles/:id', (req, res) => {
+  const id = req.params.id
+  const titleId = netflixData.filter((item) => item.show_id === +id)
+  if (!titleId) {
+    res
+      .status(404)
+      .send({ error: `No movie or tv show found with id: "${id}" found` });
+  }
+  res.json(contentWithId);
```

Comment:
I think you wanted to respond with `res.json(titleId)` or forget to create `contentWithId` ;)

Reply:
Done!